web/static/js/reducers/questionnaire.js

```diff
@@ -483,4 +549,70 @@ export const stepStoreValues = (questionnaire) => {
   }, {})
 }
 
+export const csvForTranslation = (questionnaire: Questionnaire) => {
```

Comment:
This code is nasty and full of duplicate pieces. I was supposed to refactor this after rebasing from `master`, but that turned out to be easier said than done :). 

It mostly works, except for language selection steps. 

The important things to understand are these:

-We ensure that the CSV headers start with the default lang, that's what the first few lines is about.
-We need to enclose all cells in `""` so that we can support `,` inside each column in Excel.
-We ignore language selection steps for now, out of ignorance, not intent :).
-We keep a dictionary of `exported` terms. It is just a hash of `string => bool`. The keys are always in the default lang and the values specify whether we already exported that term. That is to avoid duplicates in the output (So "Yes, Y, 1" will appear only ones although it's likely to be present in many steps).
-We intentionally skip IVR responses, because they're digits and we need them to match in all languages.